charts/knative-serving-core/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: knative-serving-core
 description: Installs Knative Serving core and CRDs.
 type: application
-version: 1.3.4
-appVersion: "v1.3.2"
+version: 1.7.4
```

Comment:
Please note that, for major/minor version upgrades, the auto upgrader won't make the PR. We need to manually update the knative-serving-istio chart where this is a dependency. cc @mbruner

Reply:
The change in https://github.com/caraml-dev/helm-charts/pull/161 should be enough right?

Reply:
Yes for the `knative-serving-istio` chart. 

We need to raise one more PR making the change in the `kserve` chart to propagate the version upgrade.